Sources/POSIX/stat.swift

```diff
@@ -10,14 +10,34 @@
 
 import libc
 
+/// A structure containing information about a file.
+public struct FileStatus {
```

Comment:
I don't think `POSIX` should contain non-POSIX things, if we want to create wrappers, or higher level things I think they should go in `Basic`, but maybe stay private.